api/client/canary/datadog.pb.go

```diff
@@ -32,9 +32,9 @@ type Datadog struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	// Whether the integration is enabled.
+	// Whether the Datadog canary service integration is enabled.
```

Comment:
`Canary service` is a bad choice of words here IMO.

We should try to be consistent with our language choices and speak a ubiquitous language around the domain of Automated Canary Analysis (ACA).

We call these `Metrics Stores` in existing documentation.